packages/calling/calling/src/CallingClient/CallingClient.ts

```diff
@@ -225,191 +100,6 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
     });
   }
 
-  /**
-   * Fetches countryCode and region of the client.
-   */
-  private async getClientRegionInfo(): Promise<RegionInfo> {
```

Comment:
AFAIK, this needs to stay in CallingClient because it'll be common for all lines

Reply:
That's correct @sreenara. We need to keep these methods in callingClient only. I understand that getMobiusServers is being invoked inside register() API and hence these needed moving due to dependency.

@meta-dipanshu-sharma  Can we try an approach where getMobiusServers can be moved out of register API and then all this logic can remain in callingClient

Let's discuss the approach over call if you need help

Reply:
Moved getMobiusServers to calling client. As we can't call async method from constructor, created a init() method which will be called while creating calling client instance. Updated the logic in calling.js file as well but for a case where initializeClients() is called from constructor(when webex object is passed), need to update the code after discussing the correct approach.

Reply:
Done